src/celeritas/field/CartMapFieldData.hh

```diff
@@ -0,0 +1,119 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/field/CartMapFieldData.hh
```

Comment:
Since including this file requires covfie, rename `CartMapFieldData.covfie.hh` (or we move all the covfie-dependent stuff into a subdir).